package-lock.json

```diff
@@ -9,207 +9,12 @@
       "version": "1.0.0",
       "license": "ISC",
       "dependencies": {
-        "@mdx-js/mdx": "^1.6.22",
```

Comment:
I think this was changed by accident. If not, sync your branch with your main branch *and* with our master branch, to get this synced between versions.

Reply:
Fixed.